src/ts/transformers/extract-details-keys.ts

```diff
@@ -1,49 +1,47 @@
-import {getHeader} from "../../helpers/har";
+import { getHeader } from "../helpers/har";
 import {
   formatBytes, formatDateLocalized, formatMilliseconds, formatSeconds, parseAndFormat, parseDate, parseNonEmpty,
   parseNonNegative, parsePositive,
-} from "../../helpers/parse";
-import {Entry, Header} from "../../typing/har";
-import {WaterfallEntry} from "../../typing/waterfall";
+} from "../helpers/parse";
+import { Entry, Header } from "../typing/har";
```

Comment:
moved file to `transformers` folder and refactor to use `Entry` instead of `WaterfallEntry`